1.10/tutorials/dcos-101/service-discovery.md

```diff
@@ -31,9 +45,14 @@ In this section, you will learn about DC/OS service discovery by exploring the d
   redis.marathon.mesos. 60  IN  A 10.0.0.43
   ```
 
-  The response tells us that the host for the `redis.marathon.mesos` is 10.0.0.43.
+  The response tells us that the host for the `redis.marathon.mesos` service is 10.0.0.43.
 
-  The A record only contains information about the host. To connect to the service, you also need to know the port. Use the following dig command to access the Service locator (SRV) DNS record, which also provides port information: `dig srv _redis._tcp.marathon.mesos`.
+  The A record only contains IP address information about the host. To connect to the service, you also need to know the port. In order to achieve this, Mesos-DNS also assigns a Service, or SRV, record for each Marathon app, which contains the port number.
```

Comment:
s/which contains/containing

the correspondence of which (app or record) seems more ambiguous otherwise